internal/business/gln_test.go

```diff
@@ -121,3 +121,21 @@ func TestGln(t *testing.T) {
 		time.Sleep(time.Second)
 	})
 }
+
+func TestCompare(t *testing.T) {
```

Comment:
test case with nil gln number is potentially missing : 
`GlnPerson{Number: nil, FirstName: ptr(" firstname "), LastName: ptr(" lastname ")}`

Reply:
GLN is the identifier... you can't have identified someone if you don't have a GLN number.